docs/docs/guides/architecture-scenarios/b2b.mdx

```diff
@@ -0,0 +1,85 @@
+
+B2b can be a simple scenario where an organization only shares one of its projects with another organization or have a more complex case where an organization is offering a portal application to all its partners with included (self)administration.
+
+<!-- This guide describes an application  -->
+## Sample scenario
+
+Octagon is a fictitious company which is used throughout this guide to explain the details and key concepts of such a B2B scenario.
+Octagon tries to solve multiple tasks in the banking field. Its portfolio includes several applications for their employees, customers, and partners. Some of which are web-based, some of which are used by machine users only.
+
+### Portal Application
+
+Octagon has a **Portal application** where its employees can access their account and list all applications they are allowed to use.
+Employees work for a department within Octagon or for Octagon itself. 
+Some of the users have enhanced features because they supervize certain teams. Those can onboard new employees and manage their roles and features.
+Target groups of the application can be split into:
+- **Employees:** users who are using the application as a starting point for their work.
+- **Supervisors:** users who are mainly using the application to manage users and their access of their department.
+- **Administrators:** this users are able to grant additional organizations or departments and elect supervisors.
+
+### Planning considerations
+
+In order to define the need of the **Portal Application** some planning considerations about organizations have to be made:
+
+- **Login and Access:** Does a user have a preset organization to login? Does the application show the default login page or does each organization use its own branding?
+- **Organizations:** Does a user have access in multiple organizations? Is a user required to use a different federated login for those organizations?
+- **Roles** Does the application need users to have specific roles assigned within their organizations? Which roles are needed to enable certain features of the application?
+
+### Login
+
+You can decide whether a organization is preselected for the login or if the user is redirected to the default login screen. You can send the user to a specific organization by defining the organization in a custom scope.
+Settings to the branding or the login options of the organization can be made from the organization section in [Console](https://console.zitadel.ch/org). 
+The behaviour of the login branding can be set in your projects detail page. You can choose the branding of the selected organization, the user resource owner, or the projects resource owner.
+
+### Organizations
+
+A user in general can be isolated to an organization, or be shared across multiple organizations. Anyways, a user should be able to use the same identity to switch between organizations.
```

Comment:
Maybe here it is not clear, that a user is always managed by one organization, but can have authroizations in multiple organizations